src/ApiTestCase.php

```diff
@@ -178,11 +178,19 @@ protected function assertResponseCode(Response $response, $statusCode)
      */
     protected function assertHeader(Response $response, $contentType)
     {
-        self::assertContains(
-            $contentType,
-            $response->headers->get('Content-Type'),
-            $response->headers
-        );
+        if (method_exists(self::class, 'assertStringContainsString')) {
+            self::assertStringContainsString(
+                $contentType,
+                $response->headers->get('Content-Type'),
+                $response->headers
+            );
+        } else {
+            self::assertContains(
```

Comment:
Using this assertion for strings is deprecated and will be removed in PHPUnit 9.